core-library/src/main/java/org/silverpeas/core/contribution/converter/DocumentFormat.java

```diff
@@ -49,7 +49,11 @@
   /**
    * The HTML format.
    */
-  html(MimeTypes.HTML_MIME_TYPE);
+  html(MimeTypes.HTML_MIME_TYPE),
+  /**
+   * The SQL format.
+   */
+  sql(MimeTypes.PLAIN_TEXT_MIME_TYPE);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS00115)